ccd-definition/AuthorisationCaseField/CareSupervision/la-solicitor.json

```diff
@@ -1758,7 +1800,35 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "courtBundleTabHeading",
```

Comment:
added the original field to AuthorisationCaseEvent/CareSupervision/DEPRECATED.json